File: easy/common_factors.go
package easy

func CommonFactors(a int, b int) int {
	count := 0
	m := max(a, b)
	for i := 1; i < m+1; i++ {
		if a%i == 0 && b%i == 0 {
			count++
		}
	}
	return count
}

File: src/easy/common-factors.ts
export const commonFactors = (a: number, b: number): number => {
    let count = 0;
    const max = (a > b ? a : b) + 1;
    for (let i = 1; i < max; i++) {
        if (a % i === 0 && b % i === 0) {
            count++;
        }
    }
    return count;
};
